docs/Factory.md

```diff
@@ -0,0 +1,104 @@
+<h1 align="center">Fastify</h1>
+
+<a name="factory"></a>
+## Factory
+
+The Fastify module exports a factory function that is used to create new
+<a href="https://github.com/fastify/fastify/blob/master/docs/Server-Methods.md"><code><b>Fastify server</b></code></a>
+instances. This factory function accepts an options object which is used to
+customize the resulting instance. This document describes the properties
+available in that options object.
+
+<a name="factory-http2"></a>
+### `http2` (Status: experimental)
+
+A truthy value that can be used to enable Node.js core's
+[HTTP/2](https://nodejs.org/dist/latest-v8.x/docs/api/http2.html) socket.
+
++ Default: `undefined` or `false`
```

Comment:
I'm expecting to see which type the http2 option is. Thruly and falsely are not clear.

Maybe typing the option is more clear

For example

If true the core http2 module is used for binding the socket.

Default: false

Reply:
I'll update these after work this evening.